app/service-catalog/services.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Service Catalog services"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+    - service-catalog
```

Comment:
```suggestion
    - service-catalog
    - gateway
```
The first one is what gets displayed in the sidebar. We should probably fix that, and display all of them if possible 🤔 

Editing to add, because now I'm not sure: Are Service Catalog services at all related to Gateway? Does `gateway` even need to be here?

Reply:
@lena-larionova Good call out. Looks like for the other Konnect app reference pages we only use the app, so I'll remove `gateway`.